CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+- the `--serve` plugin now automatically sets `NODE_ENV` environment variable to "test" instead of "testing"
+
 ## [0.2.2] - 2018-09-26
 
 - fixed `bigtest.opts` init template to use dash-cased options instead
```

Comment:
I didn't notice until now, but this (and the new change) do not have headlines.

This should have `### Fixed` above it.

And the new change should have `### Changed` above it.

Would you mind adding the heading to this old one while you're at it? 

The link to formatting is at the top of this file, but it's probably quicker if I just [drop it here](https://keepachangelog.com/en/1.0.0/)

Reply:
Sure thing. Thanks for the reference.